pwiz_tools/Shared/zedgraph/ZedGraph/Axis.cs

```diff
@@ -839,8 +839,7 @@ internal void DrawGrid( Graphics g, GraphPane pane, float scaleFactor, float shi
 			this.MinSpace = 0;
 			// Calculate the space required for the current graph assuming scalefactor = 1.0
 			// and apply the bufferFraction
-			float fixedSpace;
-			float space = this.CalcSpace( g, pane, 1.0F, out fixedSpace ) * bufferFraction;
+            float space = this.CalcSpace( g, pane, 1.0F, out _ ) * bufferFraction;
```

Comment:
Indenting looks wrong here, because this line uses spaces but all the other lines in "Axis.cs" use tabs.
I imagine this line should use tabs too, or we could replace the tabs on all of the other lines in this file.

Reply:
Noted. It's a bit jarring here so I've converted those spaces to tabs, but I don't think we want to mess with the entire directory just for formatting purposes.